dist/backdraft.js

```diff
@@ -1178,31 +1190,63 @@ _.extend(Plugin.factory, {
       var table = this;
       var filter = col.filter;
 
-      // bind focus to click event because of unbinding click from thead th when
-      // installing sort interceptors
-      $('input', head).on("click", function () {
+      // need to manually set focus because of how we unbound filtering events from the head
+      $('input', head).on('click', function() {
         this.focus();
       });
-      // update columnManager filter and ajaxUpdate dataTable when input changed
+
+      // update columnManager filter when input changed
       $('input', head).on('change', function () {
+        var filterIcon = $('.DataTables_filter_wrapper span', head);
         if (filter.type === "list") {
           if (this.checked) {
             filter[this.id] = filter[this.id] || [];
             filter[this.id].push(this.value);
+            filterIcon.removeClass("filterInactive");
+            filterIcon.addClass("filterActive");
```

Comment:
Can you re-render the template rather than tweaking classes manually like this?  The point of a template is so you can change state (like active or not) and simply re-render